src/main.ts

```diff
@@ -65,42 +65,31 @@ async function main() {
   if (productStatus.get('SOL')) {
     await solRouter.refresh_dips();
     console.log('Check SOL Position vs MM Quotes', new Date().toUTCString());
-    await solRouter.checkMMPrices();
-    // TODO: Make sure scalper gets an updated DIP position after routing
-    console.log('Run SOL Risk Manager', new Date().toUTCString());
-    solRouter.run_risk_manager();
+    solRouter.checkMMPrices();
```

Comment:
why no await?

Reply:
We want to start the 10 min window right after this. I mislabeled, the important await is really on fetchMMOrder 
https://github.com/Dual-Finance/risk-manager/blob/8c8fb94a5394fef810e3fd4be4608635fb3e403f/src/router.ts#L75